### PR TITLE
Onboarding content audit part 2 - find and fix

### PR DESCRIPTION
**Pull request by thoragudf**
Please review https://github.com/avohq/docs/pull/1186 first.

My goal here was to split up the issues view page up into "issues view", "find issues" and "fix issues" to match our messaging and make it easier to figure out how to actually find and fix the issues.

But at the same time I felt the need to create a Inspector Installation overview and consolidate two SDK overview pages into one.

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/audit/_meta**](https://docs-8fcloelur.avo-dev.app/audit/_meta) | [**/audit/branch-audit**](https://docs-8fcloelur.avo-dev.app/audit/branch-audit) |
|:-:|:-:|
|<a href="https://docs-8fcloelur.avo-dev.app/audit/_meta">Screenshot of /audit/_meta</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/kHDZEqy8r7JDqgBSy1cPvFbAQBStUGjgfv-J0OQE1eN7SiCYO63D0pxZSZzRiJ3u1-XMiGUZFLNx8xUcve-wNg.png">(view full size)</a> |<a href="https://docs-8fcloelur.avo-dev.app/audit/branch-audit">Screenshot of /audit/branch-audit</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/NxUQPfaNqubJN_uy5SEmD9puj5b5ObrjpBNg2_G4QBhYUf-OLfVrjoVDndMxwzaTDW2xg7F1OVZlPLwllBlWjw.png">(view full size)</a> |



| [**…tracking-plan-into-avo**](https://docs-8fcloelur.avo-dev.app/audit/get-tracking-plan-into-avo) | [**/audit/overview**](https://docs-8fcloelur.avo-dev.app/audit/overview) |
|:-:|:-:|
|<a href="https://docs-8fcloelur.avo-dev.app/audit/get-tracking-plan-into-avo">Screenshot of /audit/get-tracking-plan-into-avo</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ibp42zJwWFemvwvM6-uWo6qjnDnM_PnJMHhyuuov85IuzeDbuRehPepVmSKvUmkVzy2yE1OsXemxiSXAfEByHg.png">(view full size)</a> |<a href="https://docs-8fcloelur.avo-dev.app/audit/overview">Screenshot of /audit/overview</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/1L998OV2IZxA8hifNgUKL18NLY-9ClYYi6C5ydioQlhfzqfV-7G1hYQNoOjODKkzphA-_byYWjNOD602F6pICg.png">(view full size)</a> |



| [**/audit/quickstart**](https://docs-8fcloelur.avo-dev.app/audit/quickstart) | [**…sign/avo-tracking-plan**](https://docs-8fcloelur.avo-dev.app/data-design/avo-tracking-plan) |
|:-:|:-:|
|<a href="https://docs-8fcloelur.avo-dev.app/audit/quickstart">Screenshot of /audit/quickstart</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/V1aauK4BsM7vrx5pyQ6p6Sjy06BssuqmrVzqw-ueae8yOPD27A7azK4TOzvg3Z_3z5FI7GroUWKfN_5O6oKwdQ.png">(view full size)</a> |<a href="https://docs-8fcloelur.avo-dev.app/data-design/avo-tracking-plan">Screenshot of /data-design/avo-tracking-plan</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rUqWRw44H3L4TUylOtosUDW9ozIQqm37od3-jPWSFankDI9tOlMa_wqCufsmBB43UFF_mIFreBmOhz6E6rxzaw.png">(view full size)</a> |



And 14 other routes:
- [**/faqs/inspector-faq**](https://docs-8fcloelur.avo-dev.app/faqs/inspector-faq)
- [**/faqs/yes-you-can-faq**](https://docs-8fcloelur.avo-dev.app/faqs/yes-you-can-faq)
- [**/implementation/devs-101**](https://docs-8fcloelur.avo-dev.app/implementation/devs-101)
- [**/implementation/guides/start-using-inspector-with-avo-codegen**](https://docs-8fcloelur.avo-dev.app/implementation/guides/start-using-inspector-with-avo-codegen)
- [**/**](https://docs-8fcloelur.avo-dev.app/)
- [**/inspector/_meta**](https://docs-8fcloelur.avo-dev.app/inspector/_meta)
- [**/inspector/connect-inspector-to-posthog**](https://docs-8fcloelur.avo-dev.app/inspector/connect-inspector-to-posthog)
- [**/inspector/connect-inspector-to-rudderstack**](https://docs-8fcloelur.avo-dev.app/inspector/connect-inspector-to-rudderstack)
- [**/inspector/inspector-find-issues**](https://docs-8fcloelur.avo-dev.app/inspector/inspector-find-issues)
- [**/inspector/inspector-fix-issues**](https://docs-8fcloelur.avo-dev.app/inspector/inspector-fix-issues)
- [**/inspector/inspector-installation-overview**](https://docs-8fcloelur.avo-dev.app/inspector/inspector-installation-overview)
- [**/inspector/inspector-issues-view**](https://docs-8fcloelur.avo-dev.app/inspector/inspector-issues-view)
- [**/inspector/start-using-inspector**](https://docs-8fcloelur.avo-dev.app/inspector/start-using-inspector)
- [**/reference/avo-inspector-sdks/overview**](https://docs-8fcloelur.avo-dev.app/reference/avo-inspector-sdks/overview)

2 deleted routes:
- **/inspector/avo-inspector-overview**
- **/inspector/inspector-overview**

Commit 698d579371f72be820bfb21f5045acacb87ba0f2 (https://docs-8fcloelur.avo-dev.app).

File: pages/inspector/_meta.json [hunk 1]
```diff
@@ -1,22 +1,27 @@
 {
   "start-using-inspector": "Quickstart: Get started with Inspector",
+  "inspector-installation-overview": "Inspector Installation Overview",
+  "configuring-inspector-sources": "Configuring Sources",
   "-": {
     "type": "separator"
   },
-  "inspector-overview": "Inspector Overview",
```
Comment (hugihlynsson) [inline pages/inspector/_meta.json:6]: We need to remember to re-direct this url to somewhere. It's in `/next.config.js`

File: public/images/inspector/inspector-issue-details-copy-link.png
Comment (hugihlynsson) [inline public/images/inspector/inspector-issue-details-copy-link.png:1]: We should run all these png's through https://tinypng.com/

**Review by hugihlynsson — COMMENTED** [review]
Did a quick scan over the new pages and it LGTM!

File: pages/inspector/_meta.json (hunk 1 above)
Comment (thoragudf) [reply pages/inspector/_meta.json:6]: I was going to ask you about that! I removed two pages. I will add them to `/next.config.js`

Comment (thoragudf) [reply pages/inspector/_meta.json:6]: Done!

File: public/images/inspector/inspector-issue-details-copy-link.png
Comment (thoragudf) [reply public/images/inspector/inspector-issue-details-copy-link.png:1]: I thought I did that for all, but it seems like I forgot a few. Will fix!

**Pull request merged**